src/Diagnostics/Roslyn/Core/ApiDesign/DeclarePublicAPIAnalyzer.cs

```diff
@@ -110,7 +110,7 @@ public override void Initialize(AnalysisContext context)
                         return;
                     }
 
-                    var publicApiName = symbol.ToDisplayString(PublicApiFormat);
+                    string publicApiName = GetPublicApiName(symbol);
```

Comment:
Instead of  this method, can you not simply update your SymbolDisplayFormat to set [IncludeType]( http://source.roslyn.io/#Microsoft.CodeAnalysis/SymbolDisplay/SymbolDisplayMemberOptions.cs,0fc0013d966050fc) ?